src/python/pants/backend/jvm/zinc/zinc_analysis_parser.py

```diff
@@ -135,6 +142,16 @@ def _rebase_section(self, cls, header, lines_iter, outfile, rebase_mappings, jav
         if rebase_pants_home_anywhere:
           for rebased_from, rebased_to in rebase_mappings:
             rebased_line = rebased_line.replace(rebased_from, rebased_to)
+        elif rebase_pants_home_anywhere_base64:
+          val_pos = rebased_line.find(' -> ')
+          if val_pos >= 0:
+            prefix, val = rebased_line[:val_pos+4], rebased_line[val_pos+4:-1]
+          else:
+            prefix, val = '', rebased_line.rstrip()
+          val = base64.b64decode(val)
+          for rebased_from, rebased_to in rebase_mappings:
+            val = val.replace(rebased_from, rebased_to)
+          rebased_line = '{}{}\n'.format(prefix, base64.b64encode(val))
```

Comment:
Should be possible to skip re-encoding if no changes were made? Could set a variable inside the loop to mark a change.